modules/iam_users/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "project" {
+  description = "Name of the project"
+}
+
+variable "environment" {
+  description = "The environment (development/staging/production)"
+}
+
+variable "iam_roles" {
+  type        = list(tuple([string, string]))
+  description = "IAM role list with policy"
```

Comment:
We should probably convert these to maps so it's more clear.. 
Like this: 
```
list( object( {
    role    = string
    policy = string
  } ) )
```

Reply:
done

Reply:
applied